cmd/get/execution.go

```diff
@@ -146,6 +159,12 @@ func getExecutionFunc(ctx context.Context, args []string, cmdCtx cmdCore.Command
 		return err
 	}
 	logger.Infof(ctx, "Retrieved %v executions", len(executionList.Executions))
+
+	if config.GetConfig().Interactive {
+		bubbletea.Paginator(executionColumns, getCallBack(ctx, cmdCtx))
+		return nil
+	}
```

Comment:
> Can we restrict the flag to be used with only list api's and not for modification api's / on usage display that this mode is not supported for the command being used

Currently, I'm planning to add pagination to only list API functions, and they will check if -i flag is specified, other non-list API functions will not be affected even if the -i flag is set. Do you think we should explicitly inform users that interactive CLI will not be triggered if they use -i on non-list commands?

Reply:
Since we will keep adding interactive feature, I think it's fine if the flag is not yet implemented for some apis at this moment. But we should explicitly point out what apis support interactive mode when this is done.

Reply:
i Agree since we will keep implementing new changes for this mode. We can probably mention the help string that this is only support for get calls and others is no-op